system_setup/models/wslconfadvanced.py

```diff
@@ -42,67 +39,22 @@ class WSLConfigurationAdvancedModel(object):
 
     def __init__(self):
         self._wslconfadvanced = None
-        # TODO WSL: Load settings from system
 
-    def apply_settings(self, result, is_dry_run=False):
+    def apply_settings(self, result):
         d = {}
         # TODO: placholder settings; should be dynamically assgined using
         # ubuntu-wsl-integration
         d['interop_enabled'] = result.interop_enabled
```

Comment:
Same here, can we try passing dynamically result to WSLConfigurationAdvanced() directly? This avoid stuttering

Reply:
done